wagtailio/static/sass/components/_comparison-table.scss

```diff
@@ -0,0 +1,42 @@
+.comparison-table {
```

Comment:
Suggestion: add some space under this component so the next component doesn't sit too close to it. There's 100px at the large breakpoint and then i've been reducing that down to 20px for mobile.

![image](https://user-images.githubusercontent.com/31627284/184097528-80fcf5d9-9398-4efe-a001-9db8dd577706.png)